### PR TITLE
Implemented enabled flag for cron job

### PR DESCRIPTION
**Pull request by ZacZhang**

### CODE REVIEW

**Comment by ZacZhang** [comment]
I signed it!

**Comment by ZacZhang** [comment]
https://github.com/nanoPayinc/NANOPAY/issues/1471

File: src/foam/nanos/cron/Cron.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ foam.CLASS({
       name: 'scheduledTime',
       documentation: `Scheduled time to run Cron script.`,
       javaFactory: 'return getNextScheduledTime();'
+    },
+    {
+      class: 'Boolean',
+      name: 'enabled',
+      value: true,
```
Comment (kgrgreer) [inline src/foam/nanos/cron/Cron.js]: No trailing ,

File: src/foam/nanos/cron/CronScheduler.java [hunk 2]
```diff
@@ -61,6 +61,11 @@ public void run() {
           @Override
           public void put(Object obj, Detachable sub) {
             Cron cron = (Cron) obj;
+
+            if (!cron.getEnabled()) {
```
Comment (kgrgreer) [inline src/foam/nanos/cron/CronScheduler.java]: See FOAM style guide:
https://github.com/foam-framework/foam2/blob/master/doc/guides/StyleGuide.md

Comment (kgrgreer) [inline src/foam/nanos/cron/CronScheduler.java]: The enabled check can just be included in the where() expression above.

**Pull request merged**